circle.yml

```diff
@@ -1,6 +1,11 @@
 machine:
   node:
     version: 6.7.0
+dependencies:
+  pre:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
Hmm interesting thought for fixing yarn versions. Should we maybe try to bump to latest on each node bump?